packages/k-ramel/src/factoryHelpers.js

```diff
@@ -1,2 +1,19 @@
-export const keyValue = params => ({ ...params, type: 'keyValue' })
-export const simpleObject = params => ({ ...params, type: 'simpleObject' })
+// TODO: documentation
+export const types = {
```

Comment:
I don't think `types` is understandable... 
Why not `factories` or `reducers` ?

Reply:
Because I think this is more understandable that a Type can have getter, selectors, and a reducer.

This is more understandable that a `type` can be used that way: 
 - `store.data.todos.get()`

Than a reducer (redux) for whom we are used to write: 
 - `store.data.todos.get(store.getState())`

Reply:
@EmrysMyrddin @bpetetot hey ! Can I have your opinion on this ?

Reply:
Types seems to be a good term imo